### PR TITLE
Github actions

### PR DESCRIPTION
**Pull request by joakimmoller**
Moving from jenkins --> github actions and dockerhub.

### CODE REVIEW

**Comment by local-minimum** [comment]
If the existing plot is the first, I would suspect browser window size

**Comment by joakimmoller** [comment]
> If the existing plot is the first, I would suspect browser window size

aha, so the "..."  is some kind of trucation of the title and the rest of the plots is not visible in the viewport (browser window)?

**Comment by skymandr** [comment]
> If the existing plot is the first, I would suspect browser window size

This is the same problem I had last week, which for what ever reason solved itself when running again.

**Comment by skymandr** [comment]
Can I suggest adding `-v`  and see if you get better output?

**Comment by local-minimum** [comment]
Ah then someone should really add a web driver wait until thing in this test.

File: src/systemtest/test_browser.py [hunk 1]
```diff
@@ -119,7 +117,7 @@ def test_scan_get_selected(
                 "//a[@href='{0}']".format(test_ref))[0]
         href.click()
         driver.get("{}{}".format(odinapi_service, test_ref))
-        sleep(1)
+        sleep(2)
```
[the remark this answers is not in the document]
Comment (skymandr) [reply src/systemtest/test_browser.py:120]: You used this recently in another repo, right?

[the remark this answers is not in the document]
Comment (local-minimum) [reply src/systemtest/test_browser.py:120]: yes but since this is public, i'd rather not link here.

**Comment by joakimmoller** [comment]
Note that the jenkins check is not supposed to run in this branch.

**Comment by local-minimum** [comment]
> Note that the jenkins check is not supposed to run in this branch.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          python3.8 -m pip install --upgrade pip
+          pip install tox
+      - name: Javascript tests
+        run: |
+          npm install
+          npm run build
+          npm test
+      - name: Test
+        run: |
+          tox -r -- --runslow
```
Comment (local-minimum) [inline .github/workflows/ci.yml:35]: is `-r` useful? Isn't it a new machine each time?

**Review by local-minimum — APPROVED** [review]
I don't know what hotkey or thing I happened to press to accidentally close the PR but it looks good though

**Pull request merged**